openedx/core/djangoapps/credentials/models.py

```diff
@@ -1,3 +1,64 @@
 """
 Models for credentials support for the LMS and Studio.
 """
+
+from urlparse import urljoin
+
+from django.utils.translation import ugettext_lazy as _
+from django.db import models
+
+from config_models.models import ConfigurationModel
+
+
+class CredentialsApiConfig(ConfigurationModel):
+    """
+    Manages configuration for connecting to the Credential service and using its
+    API.
+    """
+    api_version_number = models.IntegerField(verbose_name=_("API Version"))
+    internal_service_url = models.URLField(verbose_name=_("Internal Service URL"))
```

Comment:
Actually we tried to have same configuration as Programs. I remember having same question for @jimabramson. The thing is having both "Internal Service URL" and "Public Service URL" same for local, development, test environments being same makes sense but when deploying to `aws` we are in a private cloud then we can have public network and one that stays inside the private network.